### ISSUE TITLE
classDiagram: Add support for typing (python)

### ISSUE THREAD

**Issue by rusty-snake** [hikerpig/pintora#266]
#### Minimal example

```
classDiagram
  class Foo {
  }

  class Bar {
    Optional[Foo] foo
  }
```

This fails. Although `Optional[Foo]` is the correct type.

I could not find a way to quote or escape it.

**Comment by hikerpig** [comment]
@rusty-snake Thank you for the issue , I'll try to fix it , it should be able to work without quotes.

**Comment by hikerpig** [comment]
This is fixed is v0.7.3
